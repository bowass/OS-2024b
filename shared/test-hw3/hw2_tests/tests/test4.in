create_list
join
insert_value 2
insert_value 3
insert_value 4
insert_value 5
insert_value 6
join
count_greater 1
join
print_list
join
delete_list
join
exit