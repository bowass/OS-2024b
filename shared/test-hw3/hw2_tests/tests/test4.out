5 items were counted
2 3 4 5 6 
